### ISSUE TITLE
Wrong handling of float values

### ISSUE THREAD

**Issue by nicofff** [kontera-technologies/graphite-api#7]
The API client is not sending float values properly to the graphite server.

Steps to reproduce:

```
require 'graphite-api'

server = GraphiteAPI.new(graphite: "localhost:2003")
server.metrics "test.float" => 0.123456
```

Dumping the whisper storage file gives the following

Archive 0 data:
0: 1431453540, 0.11999999999999999555910790149937384

The value is getting truncated to the second decimal place

Sending the same data using nc, does store the value properly:

`$ echo "test.float2 0.123456 1431453540" | nc localhost 2003`

Archive 0 data:
0: 1431453540, 0.12345599999999999629718416827017791

EDIT: Ruby version info

```
$ ruby -v
ruby 1.9.3p551 (2014-11-13 revision 48407) [x86_64-linux]
```

**Comment by nicofff** [comment]
The issue seems to be in this line:
https://github.com/kontera-technologies/graphite-api/blob/a5f21a818c4483c51ef03f67fab5afe87ce7a37f/lib/graphite-api/buffer.rb#L79

Changing the formatting from "%.2f" to "%f" caused the value to be stored correctly

**Comment by eranb** [comment]
I just uploaded a new version with your fix - `0.1.6`
Thanks a lot

**Comment by nicofff** [comment]
@eranb Thanks to you for responding so quickly!
